script.js

```diff
@@ -269,7 +269,7 @@ addClickEventListener(
 );
 addClickEventListener(
   syncExternalAccountsButton,
-  '/external-accounts/discord-sync',
+  '/external-accounts/users',
```

Comment:
Please look into tests as the endpoint has been changed

Reply:
Added a test.